src/Lachain.Core/Config/ConfigManager.cs

```diff
@@ -385,6 +387,30 @@ private void _UpdateConfigToV13()
             _SaveCurrentConfig();
         }
 
+        // version 14 of config should contain hardfork height for hardfork_12
+        private void _UpdateConfigToV14()
+        {
+            var network = GetConfig<NetworkConfig>("network") ??
+                          throw new ApplicationException("No network section in config");
+            var hardforks = GetConfig<HardforkConfig>("hardfork") ??
+                            throw new ApplicationException("No hardfork section in config");
+            hardforks.Hardfork_12 ??= network.NetworkName switch
+            {
+                "mainnet" => 0,
+                "testnet" => 0,
+                "devnet" => 0,
```

Comment:
we should use positive value for devnet and local network. otherwise we never skip verification and it halts consensus for F > 1

Reply:
it is because _degree number of coefficients for tpke key generation I guess.

Reply:
Let's set it non-zero for now,  I'll add a task to investigate it later